src/Microsoft.VisualBasic.Core/ref/Microsoft.VisualBasic.Core.cs

```diff
@@ -70,6 +71,7 @@ public sealed partial class Constants
         public const string vbCrLf = "\r\n";
         public const string vbFormFeed = "\f";
         public const string vbLf = "\n";
+        [System.ObsoleteAttribute("For a carriage return and line feed, use vbCrLf.  For the current platform's newline, use System.Environment.NewLine.")]
```

Comment:
The tool eats the `System.ComponentModel.EditorBrowsableAttribute` and `System.ObsoleteAttribute` attributes which we have to end up adding back.

1) Do we need these attributes in the reference assemblies?
2) If yes, do we need to fix the tool to keep these attributes in tact?

@ericstj, @weshaggard

Reply:
Are you sure you aren’t passing in an exclusion list?